core/pta/bcm/hw_rng.c

```diff
@@ -0,0 +1,74 @@
+// SPDX-License-Identifier: BSD-2-Clause
```

Comment:
Minor: `hw_rng` vs. `hwrng`. Currently the upstream driver uses `hwrng` so you should use this spelling here (file name, function names).

Reply:
Done